### DISCUSSION TITLE
[Esame di prova 18 dicembre 2023] Esercizio 1

### DISCUSSION THREAD

**Discussion by Elia-Belli** [sapienzastudentsnetwork/algebra#423] [Esami M-Z]
![immagine](https://github.com/sapienzastudentsnetwork/algebra2324/assets/107944476/4fbd0055-251e-4cdc-a55a-3d0362a32dfc)

**Comment by Frisbon** [comment]
braindead 
punto c piccolo errore $(abcd) = (ad)(ac)(ab) \not= (ab)(ac)(ad)$![image](https://github.com/sapienzastudentsnetwork/algebra2324/assets/84731341/df36df49-ef5f-4fd4-9609-8bb72547d9c2)
